docs/sources/shared/example.md

```diff
@@ -24,7 +24,7 @@ Below is an example from the docs, with backslashes added. The initial spaces ar
 
 ```
 \{{< docs/list >}}
-  \{{< docs/shared "manage-users/view-server-user-list.md" >}}
+  \{{< docs/shared lookup="manage-users/view-server-user-list.md" source="grafana" version="<GRAFANA VERSION>" >}}
```

Comment:
Just making sure that we're okay with this in a code block. there is also a numbered step?

Reply:
Yeah good spot. I also noticed this as part of the search and replace. I think it's fine here. This page should is just serving as an example of how shared content can work and isn't actually in use.